variables.tf

```
@@ -25,13 +25,16 @@ variable "app_url" {
   }
 }
 
-
 variable "rds_security_group_id" {
   description = "The security group attached with your RDS database."
   type        = string
-}
-
+  default     = ""
 
+  validation {
+    condition     = var.create_security_group_rule == false || length(var.rds_security_group_id) > 0
```

Im confused by this logic should it be an && operation?
```suggestion
    condition     = var.create_security_group_rule == true && length(var.rds_security_group_id) < 1
```